### ISSUE TITLE
Authentication with Signed JWT is failing

### ISSUE THREAD

**Issue by tionebsalocin** [okta/okta-sdk-python#427]
Since this commit: https://github.com/okta/okta-sdk-python/commit/bdb9838b0f83f3ff5899d403dd088693bb6743a6#diff-41c516c6f9713627b4ce8db6b845098b587d92875f254d78041b7f9478b7a6d4R61 (included in 2.9.9).

Authentication is failing because it passes the JWT (client_assertion) in the body but the API does not allow this. It must be a query param (as it was before).

Here is Okta's documentation:
https://developer.okta.com/docs/reference/rest/#get-an-access-token-from-a-signed-jwt

Would it be possible to revert this commit?

**Comment by csanders-git** [comment]
https://github.com/okta/okta-sdk-python/pull/429

**Comment by anuj530** [comment]
Yep, having the same issue here in 2.9.9 when the call works fine on 2.9.8! This is a serious problem for us right now and if someone can provide a workaround for this in the meantime, it would be great! Thank you!!

**Comment by BowlesCR** [comment]
#426 appears to be related too
